### PR TITLE
fix(help): add `--clean` to `-h`

### PR DESCRIPTION
**Pull request by clason**

### CODE REVIEW

**Review by vigoux — COMMENTED** [review]
Isn't that also equivalent to `-i NONE -i NONE --noplugins` ? Might be worth mentionning that too.

**Comment by vigoux** [comment]
Yeah that may be better. Or as you said in the comment "factory defaults" is actually a good way to explain it.

File: src/nvim/main.c [hunk 1]
```diff
@@ -2108,6 +2108,7 @@ static void usage(void)
   mch_msg(_("  -V[N][file]           Verbose [level][file]\n"));
   mch_msg("\n");
   mch_msg(_("  --api-info            Write msgpack-encoded API metadata to stdout\n"));
+  mch_msg(_("  --clean               'Factory defaults' (skip user config, user plugins, shada)\n"));
```
Comment (zeertzjq) [inline src/nvim/main.c:2111]:
```suggestion
  mch_msg(_("  --clean               \"Factory defaults\" (skip user config, user plugins, shada)\n"));
```

File: src/man/nvim.1 [hunk 2]
```diff
@@ -200,7 +200,7 @@ Skip loading plugins.
 Implied by
 .Cm -u NONE .
 .It Fl -clean
-Mimic a fresh install of Nvim. Skip loading non-builtin plugins and shada (viminfo) file.
+Start Nvim with "factory defaults" (no user config, non-builtin plugins, shada). :help --clean
```
Comment (zeertzjq) [inline src/man/nvim.1:203]: It seems `.Ic` is used for most other `:help`.

File: src/nvim/main.c [hunk 3]
```diff
@@ -2108,6 +2108,7 @@ static void usage(void)
   mch_msg(_("  -V[N][file]           Verbose [level][file]\n"));
   mch_msg("\n");
   mch_msg(_("  --api-info            Write msgpack-encoded API metadata to stdout\n"));
+  mch_msg(_("  --clean               \"Factory defaults\" (skip user config, user plugins, shada)\n"));
```
Comment (zeertzjq) [inline src/nvim/main.c:2111]:
```suggestion
  mch_msg(_("  --clean               \"Factory defaults\" (skip user config/plugins, shada)\n"));
```

Comment (clason) [reply src/nvim/main.c]: I prefer "config and plugins", if that still fits.

**Review by vigoux — APPROVED** [review]

**Comment by tankorsmash** [comment]
What does "no shada" mean?

**Pull request merged**